### ISSUE TITLE
C tests produce huge log files (and fail)

### ISSUE THREAD

**Issue by olebole** [danieljprice/giza#60]
With the new version 1.4.3, the C tests are enabled and running. However, when doing this during the Debian packaging, I get large log files:
```
-rw-r--r-- 1 1234 1234  33G  3. Dez 11:02 test-circle.log
-rw-r--r-- 1 1234 1234  33G  3. Dez 11:02 test-arrow.log
-rw-r--r-- 1 1234 1234  33G  3. Dez 11:02 test-colour-index.log
-rw-r--r-- 1 1234 1234  33G  3. Dez 11:02 test-change-page.log
```
and the tests finally somehow fail (partially with a segmentation fault). All log files contain only the repeated string
`Graphics device/type (? to see list, default /xw):`. 
Does one need to set an environment variable before running the tests?

**Comment by danieljprice** [comment]
running the tests does require user interaction (pressing enter to move to the next plotting page), so in the GitHub actions workflow I run the tests with the following line:

```
yes "" | make check
```
which presses Enter in response to any prompting. My guess is the large log files are because it either does not get a response to the prompt, or cannot launch an X-window. In the actions workflow I also allowed window launching using Xvfb. Full details here:

https://github.com/danieljprice/giza/blob/master/.github/workflows/build.yml

To get the actions workflow to pass, I had to disable a couple of the tests that require mouse clicks, so the tests that are there just require window launching and pressing Enter in the terminal.
